xds/src/main/java/io/grpc/xds/ClusterResolverLoadBalancer.java

```diff
@@ -342,6 +345,8 @@ void shutdown() {
     private final class EdsClusterState extends ClusterState implements EdsResourceWatcher {
       @Nullable
       private final String edsServiceName;
+      private Map<Locality, String> localityPriorityNames = new HashMap<>();
```

Comment:
`Collections.emptyMap()` would make it a bit more obvious the reference is modified instead of the map itself.